com.github.bitseather.weather.json

```diff
@@ -0,0 +1,168 @@
+            ],
+            "build-options": {
+                "env": {
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_GIRDIR": "/app/share/gir-1.0",
+                    "PKG_CONFIG_GOBJECT_INTROSPECTION_1_0_TYPELIBDIR": "/app/lib/girepository-1.0"
+                }
+            },
+            "sources": [{
+                "type": "archive",
+                "url": "https://launchpad.net/libdbusmenu/16.04/16.04.0/+download/libdbusmenu-16.04.0.tar.gz",
+                "sha256": "b9cc4a2acd74509435892823607d966d424bd9ad5d0b00938f27240a1bfa878a"
+            }]
+        },
+        {
+            "name": "libindicator",
+            "config-opts": ["--with-gtk=3", "--disable-tests"],
+            "sources": [{
+                    "type": "archive",
+                    "url": "https://launchpad.net/libindicator/12.10/12.10.1/+download/libindicator-12.10.1.tar.gz",
+                    "sha256": "b2d2e44c10313d5c9cd60db455d520f80b36dc39562df079a3f29495e8f9447f"
+                },
+                {
+                    "type": "shell",
+                    "commands": [
+                        "sed 's/gtk_icon_info_free/g_object_unref/' -i libindicator/indicator-image-helper.c",
+                        "sed 's/LIBS=\"-lm  $LIBS\"/LIBS=\" -lm  $LIBS\"/g' -i configure",
+                        "sed 's/LIBM=\"-lm\"/LIBM=\" -lm\"/g' -i configure"
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "libappindicator",
+            "config-opts": [
+                "--with-gtk=3",
+                "--disable-gtk-doc-html",
+                "--disable-mono-test",
+                "--disable-static",
+                "--disable-tests"
+            ],
+            "rm-configure": true,
+            "sources": [{
+                    "type": "archive",
+                    "url": "https://launchpad.net/libappindicator/12.10/12.10.0/+download/libappindicator-12.10.0.tar.gz",
+                    "sha256": "d5907c1f98084acf28fd19593cb70672caa0ca1cf82d747ba6f4830d4cc3b49f"
+                },
+                {
+                    "type": "patch",
+                    "path": "no-python.patch"
+                },
+                {
+                    "type": "script",
+                    "commands": [
+                        "autoreconf -vfi"
+                    ],
+                    "dest-filename": "autogen.sh"
+                }
+            ]
+        },
+        {
+            "name": "weather",
+            "buildsystem": "meson",
+            "config-opts": ["--buildtype=release"],
+            "sources": [{
+                    "type": "archive",
+                    "url": "https://github.com/bitseater/weather/archive/0.5.3.tar.gz",
+                    "sha256": "36849f4220e75296610ebb469df243db3e36a1549dd184e4364e9b92282921f8"
+                },
+                {
+                    "type": "patch",
+                    "path": "meson-fix.patch"
```

Comment:
Was it proposed upstream?

Reply:
Yeah, and already merged along with some other fixes! Just waiting for a new release